lib/storage/src/content_manager/toc.rs

```diff
@@ -312,14 +317,16 @@ impl TableOfContent {
         }
     }
 
-    pub async fn get_collection(
-        &self,
+    pub async fn get_collection<'a>(
```

Comment:
Clion sais this lifetime annotation is redundant

Reply:
It's probably a false positive. Initially I introduced it only because compiler could not derive it. But I will check.

Reply:
Yes, rechecked. The lifetime is required there.